pkg/types/baremetal/platform.go

```diff
@@ -33,10 +24,16 @@ type Platform struct {
 	// Default is qemu:///system
 	LibvirtURI string `json:"libvirtURI,omitempty"`
 
-	// IronicURI is the identifier for the Ironic connection.  It must be
-	// reachable from the host where the installer is run.
+	// ClusterProvisioningIP is the IP on the provisioning host private/provisioning
+	// network for the host running the installer, can be used to cache some
+	// downloaded content e.g RHCOS/IPA images
 	// +optional
-	IronicURI string `json:"ironicURI,omitempty"`
+	ClusterProvisioningIP string `json:"provisioningHostIP,omitempty"`
```

Comment:
doesn't provide any guidelines on if this IP can be part of the machine CIDR? or the pod CIDR etc..

Reply:
Ack, yeah it definitely cannot be part of those CIDRs so some validation would be a good idea.

I had hoped to change this to something like ProvisioningCidr in due course, but we've not quite got all the templating in place to enable that yet.

Reply:
I raised #2210 so this validation can perhaps be a follow-up PR?